core/templates/dev/head/editor/exploration_history.html

```diff
@@ -18,8 +18,7 @@
     <h3>List of Changes</h3>
     <div ng-repeat="versionNumber in snapshotOrderArray" class="row">
       <div class="col-sm-2 col-md-2 col-lg-2">
-        <input type="radio" name="compareVer1" ng-model="compareVersions.selectedVersion1" ng-value="versionNumber" ng-change="changeCompareVersion(versionNumber, 'selectedVersion1')" class='protractor-test-history-v1-selector'>
-        <input type="radio" name="compareVer2" ng-model="compareVersions.selectedVersion2" ng-value="versionNumber" ng-change="changeCompareVersion(versionNumber, 'selectedVersion2')" class='protractor-test-history-v2-selector'>
+        <input type="checkbox" name="compareVer1" ng-model="versionArr" ng-click="submitVersion($event, versionNumber)" class='protractor-test-history-v1-selector'>
```

Comment:
The 'protractor-test-history-v1-selector' (and the v2-selector in the old version) is used in integration tests. (See tests/protractor/editorAndPlayer.js and tests/protractor_utils/editor.js for the code, and https://github.com/oppia/oppia/wiki/Writing-Integration-Tests for an overview of the tests.) The integration tests also need to be changed to use checkboxes instead of radio buttons. (The relevant code is near the end of tests/protractor_utils/editor.js)